src/Sarif.Viewer.VisualStudio.ResultSources.DeveloperCanvas.Core/Services/DevCanvasResultSourceService.cs

```diff
@@ -131,13 +135,16 @@ public class DevCanvasResultSourceService : IResultSourceService
                 return (int)getOptionStateCallback("DevCanvasServer");
             };
 
-            accessor = new DevCanvasWebAPIAccessor(serverOptionAccess);
+            AuthState.Initialize();
+            AuthManager authManager = new AuthManager(SetOptionStateCallback);
+            accessor = new DevCanvasWebAPIAccessor(serverOptionAccess, authManager);
+
         }
 
         /// <inheritdoc/>
         public System.Threading.Tasks.Task InitializeAsync()
         {
-            DevCanvasTracer.WriteLine($"Initializing {nameof(DevCanvasResultSourceService)}. Version 8/2");
+            DevCanvasTracer.WriteLine($"Initializing {nameof(DevCanvasResultSourceService)}. Version 9/7");
```

Comment:
Manual insertion of the version date - I assume is a temporary stop gap?  Want to add a quick comment with TODO or some way to track in code if need?

Reply:
This has just been something I've been doing due to sometimes the installer not installing the correct ver / teams uploading a copy of the old vsix rather than the new one when I try to distribute to team members.
Planning on removing when we merge into the main branch, I'll add a TODO here just in case.